### PR TITLE
App/writefreely

### PR DESCRIPTION
**Pull request by lukasitaly**
[WriteFreely](https://writefreely.org/)

- [x]  Port is not in use already
- [x]  Followed the ["Adding a new app"](https://runtipi.io/docs/contributing/adding-a-new-app) procedure
- [x]  Logo looks good in the app store
- [x] Tested

## Summary by CodeRabbit

- **New Features**
	- Added WriteFreely app to the Tipi App Store, increasing total available apps to 264.
	- Launched a Docker Compose configuration for the WriteFreely service, facilitating easy deployment.

- **Documentation**
	- Updated README.md to include WriteFreely app details.
	- Created a new description.md file outlining WriteFreely’s capabilities and setup process, including admin account creation and key features.

### CODE REVIEW

File: apps/write-freely/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    container_name: write-freely
+    environment:
+      TZ: ${TZ}
+      PUID: 1000
+      PGID: 1000
+    ports:
+      - 40:70/tcp
+      - ${APP_PORT}:8080/tcp
+    volumes:
+      - ${APP_DATA_DIR}/data/config:/mnt/config
```
[the remark this answers is not in the document]
Comment (lukasitaly) [reply apps/write-freely/docker-compose.yml:13]: Not needed - see https://hub.docker.com/r/nephatrine/write-freely

File: apps/write-freely/metadata/description.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+&nbsp;
+<p align="center">
+	<a href="https://writefreely.org"><img src="https://writefreely.org/img/writefreely.svg" width="350px" alt="WriteFreely" /></a>
+</p>
+<hr />
+&nbsp;
+
+WriteFreely is a clean, minimalist publishing platform made for writers. Start a blog, share knowledge within your organization, or build a community around the shared act of writing.
+
+![](https://i.snap.as/cQZxXoz.png)
```
[the remark this answers is not in the document]
Comment (lukasitaly) [reply apps/write-freely/metadata/description.md]: Done

**Comment by steveiliop56** [comment]
Hello @lukasitaly,

Thank you for the pull request! I am bit confused concerning your port number choices : ). Why did you choose such a big port like 11140 and not something like 8574. Also I see that the container is using port 70 too, a quick google search says that is for a protocol called the gohper protocol, maybe we should keep port 70 instead of 40? Apart from these everything looks ok and the app works so we can merge.

**Comment by lukasitaly** [comment]
Hey @steveiliop56, thanks for the comment!

I didn't realize I was using port 40, and I'm not sure why I changed it. I had been using the same YAML file up until now, so I just copied it over. I also chose port 11140 for the same reason - I prefer using high ports because they're less likely to be used by other services or Runtipi. Since I checked and saw that another app in the Tipi store wasn't using it, I decided to stick with it.

I changed the ports as you suggested and now everything should be in order :)

**Comment by steveiliop56** [comment]
Cool! I am merging!

**Review by steveiliop56 — APPROVED** [review]

**Pull request merged**